benefits/static/css/styles.css

```diff
@@ -150,6 +150,16 @@ h1 {
   padding-top: 70px;
 }
 
+/* First paragraph immediately after an H1 gets padding-top
+h1 + p:first-of-type {
+  padding-top: 24px;
+} */
+
+/* Last paragraph after an H1 gets padding bottom
+h1 ~ p:nth-last-of-type(1) {
+  padding-bottom: 24px;
+} */
```

Comment:
Did these get commented out during testing?

Reply:
Oops my git was slow and didn't push these up: https://github.com/cal-itp/benefits/pull/1041/commits/af3d1fce09dd74f21c2a00175046c54f74b9d940